File: windows/RNFS-vnext/ReactPackageProvider.cs
using Microsoft.ReactNative.Managed;
using Microsoft.ReactNative;

namespace RNFSvnext
{
    public sealed class ReactPackageProvider : IReactPackageProvider
    {
        public void CreatePackage(IReactPackageBuilder packageBuilder)
        {
            packageBuilder.AddAttributedModules();
        }
    }
}

File: tmp-original-windows/RNFS/ReactPackageProvider.cpp
#include "pch.h"
#include "ReactPackageProvider.h"
#include "ReactPackageProvider.g.cpp"

#include "RNFSManager.h"

using namespace winrt::Microsoft::ReactNative;

namespace winrt::RNFS::implementation {

void ReactPackageProvider::CreatePackage(IReactPackageBuilder const &packageBuilder) noexcept {
  AddAttributedModules(packageBuilder);
}

} // namespace winrt::RNFS::implementation
